Update PingSource docs and examples to use v1beta2 (#3046)
knative/docs: docs/eventing/samples/ping-source/index.md

{{< /tab >}}
{{< /tabs >}}

## (Optional) Create a PingSource with binary data

> Sometimes you might want to send binary data that cannot be directly serialized in yaml to downstream. This can be achieved by using `dataBase64` as the payload, as the name suggests, `dataBase64` should carry data that is base64 encoded.

Reviewer: ```suggestion
Sometimes you may want to send binary data, which cannot be directly serialized in yaml, to downstream. This can be achieved by using `dataBase64` as the payload. As the name suggests, `dataBase64` should carry data that is base64 encoded.
```

Please note that `data` and `dataBase64` cannot co-exist.

{{< tabs name="create-source" default="By YAML" >}}
{{% tab name="By YAML" %}}

> Use following command to create the event source with binary data from STDIN:

Reviewer: ```suggestion
Use the following command to create the event source with binary data from STDIN:
```

```shell
cat <<EOF | kubectl create -f -
apiVersion: sources.knative.dev/v1beta2
kind: PingSource
metadata:
  name: test-ping-source-binary
spec:
  schedule: "*/2 * * * *"
  contentType: "text/plain"
  dataBase64: "ZGF0YQ=="
  sink:
    ref:
      apiVersion: serving.knative.dev/v1
      kind: Service
      name: event-display
EOF
```
{{< /tab >}}

{{% tab name="By filename" %}}

> Use following command to create the event source from the `ping-source-binary.yaml` file:

Reviewer: ```suggestion
Use the following command to create the event source from the `ping-source-binary.yaml` file:
```
